doc/helper-guide/odp.dox

```diff
@@ -16,6 +16,10 @@
  *
  *  @image html overview.svg
  *
+ * ODP must be initialized before calling helper functions, unless stated
+ * otherwise. Pre-processor macros may be invoked independently of ODP
+ * initialization.
```

Comment:
As discussed, this same clause could also be added to some helper API header (for example `helper/include/odp/helper/odph_api.h`).

Reply:
When we discussed this, I liked the idea of having this in a header. But putting this in the `@file` documentation of odph_api.h would put in https://opendataplane.github.io/odp/helper-guide/odph__api_8h.html#details, which is pretty well hidden. So it's also needed in odp.dox, so that it appears on the main page https://opendataplane.github.io/odp/helper-guide/. So we would have two copies of the paragraph in two different places, which is not ideal, but maybe it's fine?

Reply:
Yep, it's not ideal but better to have the documentation in both places.

Reply:
In v3.